packages/gestalt/src/ModuleExpandable.js

```diff
@@ -23,9 +25,13 @@ export default function ModuleExpandable({
   id,
   accessibilityExpandLabel,
   accessibilityCollapseLabel,
+  extExpandedId,
+  setExtExpandedId,
   items,
 }: Props): Node {
-  const [expandedId, setExpandedId] = useState(-1);
+  const [localExpandedId, setLocalExpandedId] = useState(null);
+  const expandedId = extExpandedId || localExpandedId;
```

Comment:
can we simplify the name?

`expandedId` and `setExpandedId`

we can differentiate it internally, but keep the API shorter so it's easier to read

Reply:
changed the external param
extExpandedId ->  expandedId
setExtExpandedId -> setExpandedId